cl_tf2/src/cl-transforms-plugin/ros-msg-conversions.lisp

```diff
@@ -51,6 +58,12 @@
     (cl-transforms:make-transform
      (msg->3d-vector translation) (msg->quaternion rotation))))
 
+(defun msg->pose (msg)
```

Comment:
This function (and its counterpart, `pose->msg`) are right now only implemented in `cl-tf`, which we want to get rid of as far as I understand. That's why I re-implemented them here.